pkg/auth/authTokensGet_test.go

```diff
@@ -121,10 +148,69 @@ func TestInvalidPathReturnsError(t *testing.T) {
 	console := utils.NewMockConsole()
 
 	//When
-	err := GetTokens(apiClient, console)
+	err := GetTokens(apiClient, console, "")
 
 	//Then
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "GAL1146E")
 	assert.Contains(t, err.Error(), "Could not get list of tokens from API server")
 }
+
+func TestMissingLoginIdFlagReturnsBadRequest(t *testing.T) {
+	//Given...
+	serverState := "missingLoginId"
+	server := NewAuthTokensServletMock(t, serverState)
+	apiClient := api.InitialiseAPI(server.URL)
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `GAL1155E: The id provided by the --id field cannot be an empty string.`
+
+	//When
+	err := GetTokens(apiClient, console, "   ")
+
+	//Then
+	assert.NotNil(t, err)
+	assert.Equal(t, expectedOutput, err.Error())
+}
+
+func TestLoginIdWithSpacesReturnsBadRequest(t *testing.T) {
+	//Given...
+	serverState := "invalidLoginIdFlag"
+	server := NewAuthTokensServletMock(t, serverState)
+	apiClient := api.InitialiseAPI(server.URL)
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `GAL1157E: 'galasa admin' is not supported as a valid value. Valid value should not contain spaces. A value of 'admin' is valid but 'galasa admin' is not.`
```

Comment:
The last sentence in this error can be removed since we're already saying that spaces can't be included in login IDs.

Also instead of `Valid value`, maybe `Valid usernames`.
Instead of `Valid value should not contain spaces`, maybe something like `Usernames should not contain spaces`

(again, feel free to replace `username` with `login ID` if you want!)